About and Contact page

kimballh opened the pull request:
Adds links on left of navbar that go to "About" and "Contact" information pages.

pjtatlow commented on site/ui/components/shared/Contact.vue:
```diff
@@ -0,0 +1,47 @@
+        // font-weight: bold;
+    }
+    p {
+        font-size: 2em;
```
I don't know if it's just me, but I have a hard time reading this with the font so big! @srp33 can you take a look at the size and let me know if that is what you want?

srp33 replied:
Sorry for being slow, but what should I do to see how it currently looks?

kimballh replied:
@srp33, you can preview the changes on this branch when you are on campus at this url: http://10.38.32.235:8080/contact Let me know if you want me to add/remove anything or change wording.

pjtatlow commented:
@kimballh this looks great! Other than the font size the only thing I'd like to change is move the `Contact` and `About` pages into the `main` directory rather than `shared`. I've tried to only put components in shared that are actually used across multiple pages. Although now that I think about it, there's a few things that don't belong in there too! Can you also move `Error404`, `Login` and `Navbar` too? Thanks!

pjtatlow commented:
Just thought of one more thing. When you're on the `About` or `Contact` page, we don't want the breadcrumbs up top to say "Select Dataset" because they're not on a page where they can do that. Can you fix this as well?

pjtatlow commented on site/ui/components/shared/Navbar.vue:
```diff
@@ -15,6 +15,10 @@
         <span>Geney</span>
       </b-navbar-brand>
+      <b-navbar-nav class="above-breadcrumbs ml-auto">
+        <b-nav-item href="/about" class="pr-2">About</b-nav-item>
```
We should make this a `router-link` so that the whole page doesn't reload when we go to `/about`. Let me know if you have questions about this.

pjtatlow commented on site/ui/components/shared/Navbar.vue:
```diff
@@ -15,6 +15,10 @@
       </b-navbar-brand>
+      <b-navbar-nav class="above-breadcrumbs ml-auto">
+        <b-nav-item href="/about" class="pr-2">About</b-nav-item>
+        <b-nav-item href="/contact">Contact</b-nav-item>
```
Same here.

kimballh commented:
I got rid of the breadcrumbs on those static pages, made router-links, adjusted the font size and margins a bit and moved the components you mentioned into main except for navbar, isn't that a shared component? Let me know if you notice anything else, thanks for the feedback!

kimballh commented:
I also removed the background from one of the logos and created a favicon that looks nice

pjtatlow commented on site/ui/components/shared/Navbar.vue:
```diff
@@ -126,6 +126,7 @@ export default {
       }
       return [];
     },
+    staticPages: (Array) => ['About', 'Contact', 'DatasetNotFound'],
```
Just so you know, this is actually a function that returns an array, rather than just being an array. I'll fix it though!

pjtatlow replied:
All I ended up doing was declaring a property in `data` rather than a getter function in `computed`. This way it doesn't have to call a function every time to get the list of `staticPages`.
